src/components/WorkAreaDock/TheoryContent.tsx

```diff
@@ -0,0 +1,67 @@
+export function TheoryContent(): JSX.Element {
+  return (
+    <div
+      style={{
+        margin: "10px",
+        padding: "5px",
+        border: "4px solid #EEEEEE",
+        overflowY: "scroll",
+        height: "50vh",
+      }}
+    >
+      <h2>Monte Carlo Tree Search</h2>
+      <p>
+        Monte Carlo Tree Search is a heuristic search algorithm. It is used in decision processes
+        e.g. in the AlphaGo playing by Artificial Intelligence. In cases like Go, there are too many
+        options to calculate reward for every path. MCTS enables AI to predict the most valuable
```

Comment:
Myślę, że w drugiej wersji można by było podać `MCTS` w nawiasach przy `Monte Carlo Tree Search`. 
Wiem, że i tak wiadomo o co chodzi, ale u mnie czasami bywa tak, że nie mogę od razu wpaść na to co oznacza jakiś skrót (nawet jeśli jest oczywisty)